metrics-dropwizard/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'scala'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(':usi-metrics')
+    compile group: 'io.dropwizard.metrics', name: 'metrics-core', version: '4.0.2'
+    compile group: 'io.dropwizard.metrics', name: 'metrics-jvm', version: '4.0.2'
```

Comment:
What do you think about sorting alphabetical: `com.github...`, `io.dropwizard...`?

Reply:
alphabetical by the group name? artifact name?

Reply:
First group name then artifact name.

Reply:
Have fun enforcing this rule across the project :D I'll do that but just because it's you!